pkg/api/user.go

```diff
@@ -14,3 +17,17 @@ var CurrentUser = func(client *gitlab.Client) (*gitlab.User, error) {
 	}
 	return u, nil
 }
+
+var UserByName = func(client *gitlab.Client, name string) (*gitlab.User, error) {
+	opts := &gitlab.ListUsersOptions{Username: gitlab.String(name)}
+	users, _, err := apiClient.Users.ListUsers(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(users) != 1 {
+		return nil, errors.New(fmt.Sprintf("failed to find user by name : %s", name))
```

Comment:
I would prefer you use `fmt.Errorf()` instead of having `errors.New(fmt.Sprintf())` to reduce complexity

Reply:
Oops. just realized you fixed this with cd41e36 👍